driver.go

```diff
@@ -3,25 +3,29 @@ package main
 import (
 	"errors"
 	"github.com/docker/go-plugins-helpers/volume"
+	"os"
+	"syscall"
 )
 
+// The base directory of docker-on-top, where all the overlays' internals will be saved
+const dotBaseDir string = "/var/lib/docker-on-top/"
+
 // Driver contains internal data for the docker-on-top volume driver. It does not export any fields
 type Driver struct {
-	// TODO: some fields will be here. For now it's just a stub implementation
-	volumesCreated map[string]bool // In this implementation the value does not mean anything, only the key
+	volumesCreated  map[string]volume.Volume
+	overlaysCreated map[string]string
```

Comment:
The objects in these maps seem to be created and deleted together. Convert to a map of struct!